colossus-metrics/src/main/scala/colossus/metrics/StatReporter.scala

```diff
@@ -64,6 +62,19 @@ class MetricReporter(intervalAggregator : ActorRef, config: MetricReporterConfig
     }
   }
 
+  private def filterMetrics(m : MetricMap) : RawMetricMap = {
+    filters match {
+      case MetricReporterFilter.All => m.toRawMetrics
+      case MetricReporterFilter.None => Map()
```

Comment:
Yea..i I figured as much..seems totally weird.  I just put it in in the even that they just wanted to quickly silence metric reporting.  I guess they could just comment out that line.  I can remove that case.